fgpyo/util/metric.py

```diff
@@ -322,3 +338,34 @@ def fast_concat(*inputs: Path, output: Path) -> None:
             io.write_lines(
                 path=output, lines_to_write=list(io.read_lines(input_path))[1:], append=True
             )
+
+    @staticmethod
+    def _read_header(
+        reader: io.Reader,
+        comment_char: str = DEFAULT_HEADER_COMMENT_CHAR,
```

Comment:
I would prefer to remove defaults from non-public methods to make sure that the values are being vectored appropriately.

Reply:
👍